dali/test/python/operator/test_split_merge.py

```diff
@@ -0,0 +1,146 @@
+# Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
```

Comment:
What about nested split-merges? Are they allowed? If so, maybe it would be good to test them?

Reply:
There is nothing specific that shouldn't allow nesting. I will cover the nesting use-case when we land the if-statement, that would be more interesting case, here I don't think we need it.